ui/src/forms/metadata.ts

```diff
@@ -0,0 +1,7 @@
+import { dataset, literal, quad } from '@rdf-esm/dataset'
+import { editor } from '@/forms/bulma'
+import { rdfs } from '@tpluscode/rdf-ns-builders'
+
+export const Metadata = dataset([
+  quad(editor.RadioButtons, rdfs.label, literal('Radio buttons'))
+])
```

Comment:
DASH editor metadata is RDF

Reply:
What is this metadata?

Reply:
Right now it's just labels and the editor type (can be `dash:SingleEditor` or `dash:MultiEditor`)